Dijkstra_Algorithm/Dijkstra_Algorithm.dart

```diff
@@ -0,0 +1,153 @@
+/*
+ Djikstra's algorithm (named after its discover, E.W. Dijkstra) solves the
+ problem of finding the shortest path from a point in a graph (the source)
+ to a destination.
+ It turns out that one can find the shortest paths from a given source to all
+ points in a graph in the same time, hence this problem is sometimes called
+ the single-source shortest paths problem.
+*/
+import 'dart:io';
+
+var INT_MAX = 9223372036854775807;
+
+int minDistance(dist, visited, n)
+{
+    int min = INT_MAX, min_index;
+
+    for (var v = 0; v < n + 1; v++)
+    {
+        if (( visited[v] == false ) && ( dist[v] <= min ))
+        {
+            min = dist[v];
+            min_index = v;
+        }
+    }
+
+    return min_index;
+}
+
+void printsol(dist, n)
+{
+    print('Vertex \t\t Distance from Source\n');
+
+    for (var i = 0; i < n + 1; i++)
+    {
+        print('${i} \t\t ${dist[i]}\n');
+    }
+}
+
+void dijkstra(graph, src, n)
+{
+    var dist = new List(n + 1);
+
+    var visited = new List(n + 1);
+
+    for (var i = 0; i < n + 1; i++)
+    {
+        dist[i] = INT_MAX;
+        visited[i] = false;
+    }
+
+    dist[src] = 0;
+
+    for (var count = 0; count < n; count++)
+    {
+
```

Comment:
remove this line

Reply:
Made necessary changes @asha15  please review.